src/langsmith/manage-organization-by-api.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Manage your organization using the API
-sidebarTitle: Manage your organization using the API
+sidebarTitle: Manage organization using the API
```

Comment:
Not sure how tight we are on characters but if "organizations" fits I might suggest using that

Reply:
yep, sounds better!